.env.test

```diff
@@ -49,3 +49,10 @@ FEEDBACK_ERROR_URL=https://reftracker.nla.gov.au/reft100.aspx?key=Report_an_erro
 
 # Patron Keycloak migration
 PATRON_UPGRADE_URL=https://test.nla.gov.au/patron/upgrade
+
+# Copyright
+COPYRIGHT_SERVICE_URL=https://test.nla.gov.au/copyright/
+COPYRIGHT_CONTACT_URL=https://test.nla.gov.au/contact-us
+
+# Copies Direct
+COPIES_DIRECT_URL=https://test.nla.gov.au/copies-direct
```

Comment:
shouldnt this be copiesdirect-test.nla.gov.au?

Reply:
I don't actually want to hit the real URL because these tests are running on GitHub and don't have access to the lower end. Another reason is that this code is public, so revealing real URLs is not something I want to do.

Reply:
ah,  yeah that explains things, thanks.